wordpress-mobile/WordPress-Android
# Do not replace the image drawable unless the bitmap is non-empty

Fixes #5045.

To test: 
1. Make sure Aztec is selected as the default editor in the settings
2. Open a post with images
3. Verify the image placeholder is shown and eventually is replaced by the actual image
4. Go back to the post list, then go back to the same post
5. Verify the image is loaded immediately from cache
